libs/ui-lib/lib/ocm/services/types.ts

```diff
@@ -7,7 +7,7 @@ import {
 
 export type ClusterDetailsUpdateParams = Pick<
   V2ClusterUpdateParams,
-  'name' | 'baseDnsDomain' | 'pullSecret'
+  'name' | 'baseDnsDomain' | 'pullSecret' | 'platform'
 >;
 
 export type ClusterCreateParamsWithStaticNetworking = ClusterCreateParams & {
```

Comment:
(Not for this line)
Are we using `useClusterSupportedPlatforms`?

Are we using the response from  `${ClustersAPI.makeBaseURI(clusterId)}/supported-platforms``?

Doesn't seem that with the new implementation, we're validating if the platform is supported according to this endpoint. Therefore, should it be removed too?

Reply:
We need to clarify this point @jkilzi

Reply:
@celdrake  useClusterSupportedPlatforms is used in some cases (in NetworkConfiguration and in ClusterPlatformIntegrationHint). Is not used more in Host Discovery step.